R/discovery_prediction.R

```diff
@@ -19,7 +37,7 @@ NULL
 #' num_signatures = 3, method = "lda", seed = 12345, nstart = 1)
 #' @export
 discover_signatures <- function(musica, table_name, num_signatures,
-                                method="lda", seed = 1, nstart = 1,
+                                method="lda", seed = 1, nstart = 10,
```

Comment:
```suggestion
                                method="lda", seed = 1, nstart = 1,
```

Reply:
nstart = 10 will increase runtime significantly; do we have something to suggest this is necessary? I would prefer it keep it at the default users will expect.